src/common/transformations/include/transformations/common_optimizations/eliminate_unsqueeze_gather.hpp

```diff
@@ -37,8 +37,15 @@ class ngraph::pass::EliminateUnsqueezeGather : public ngraph::pass::MatcherPass
  * Unsqueeze makes it a 1D tensor
  */
 
-class ngraph::pass::EliminateGatherUnsqueeze : public ngraph::pass::MatcherPass {
+class ov::pass::EliminateGatherUnsqueeze : public ov::pass::MatcherPass {
 public:
     OPENVINO_RTTI("EliminateGatherUnsqueeze", "0");
     EliminateGatherUnsqueeze();
 };
+
+namespace ngraph {
```

Comment:
Why do you need this aliases for internal API?

Reply:
Corresponding JIRA ticket guides to split this task into two separate PR's. I'm going to add third PR of renaming `ngraph_ops` directory to `ov_ops` separated in order to have easier review.

Reply:
The other reason is that some openvino_contrib plugins depend of those. To remove `ngraph::pass` with `ov::pass` in those plugins I need to have both for a while, otherwise CI's will fail.